pagopa/io-app
# feat: [PE-805] CGN update cta on card remove action

## Short description
The changes focus on updating the text for deactivating the `Carta Giovani Nazionale`

## List of changes proposed in this pull request
- Update localizations
- Update alert render view

## How to test
- Add CGN to the wallet
- Check the remove button text
- Try to remove the CGN card
- Check the alert text and then continue
- Check the toast message in both positive and negative case
